data/Luminometrics/20161115-high-throughput-test-96-wells.csv

```diff
@@ -1 +1,51 @@
-Well,SampleID,Replicate,Well Type,Date,Luminescence
13,L1,1,96,20161115,809.082
14,L2,1,96,20161115,-14.3707
15,L4,1,96,20161115,-464.008
16,L6,1,96,20161115,-40.9539
17,L7,1,96,20161115,422.306
18,L11,1,96,20161115,-370.145
19,L12,1,96,20161115,-496.582
20,L16,1,96,20161115,526.813
21,Mock,1,96,20161115,-286.388
22,Mock,2,96,20161115,-49.1675
23,L1,2,96,20161115,3908.95
24,L2,2,96,20161115,-264.671
25,L4,2,96,20161115,364.933
26,L6,2,96,20161115,-84.9508
27,L7,2,96,20161115,-347.168
28,L11,2,96,20161115,74.1227
29,L12,2,96,20161115,-204.713
30,L16,2,96,20161115,48.9294
31,PHH,1,96,20161115,621.501
32,PHH,2,96,20161115,805.512
33,L17,1,96,20161115,-190.471
34,L18,1,96,20161115,-307.878
35,L19,1,96,20161115,-291.054
36,L20,1,96,20161115,23.3205
37,L22,1,96,20161115,-282.172
38,L23,1,96,20161115,-303.39
39,L24,1,96,20161115,-475.741
40,L26,1,96,20161115,482.678
41,PCD,1,96,20161115,1096020
42,PCD,2,96,20161115,1028630
43,L17,2,96,20161115,60.8281
44,L18,2,96,20161115,332.344
45,L19,2,96,20161115,3863.74
46,L20,2,96,20161115,2818.24
47,L22,2,96,20161115,582.08
48,L23,2,96,20161115,583.117
49,L24,2,96,20161115,-137.397
50,L26,2,96,20161115,545.602
51,Vic,1,96,20161115,216380
52,Vic,2,96,20161115,91222.6
53,1:02,1,96,20161115,4392890
54,1:02,2,96,20161115,4195230
55,1:04,1,96,20161115,549184
56,1:04,2,96,20161115,972067
57,1:08,1,96,20161115,627501
58,1:08,2,96,20161115,930836
59,1:16,1,96,20161115,728666
60,1:16,2,96,20161115,485235
61,1:32,1,96,20161115,310613
62,1:32,2,96,20161115,90436.8
\ No newline at end of file
+Well,SampleID,Replicate,Well Type,Date,Luminescence
+13,L1,1,96,20161115,809.082
+14,L2,1,96,20161115,-14.3707
+15,L4,1,96,20161115,-464.008
+16,L6,1,96,20161115,-40.9539
+17,L7,1,96,20161115,422.306
+18,L11,1,96,20161115,-370.145
+19,L12,1,96,20161115,-496.582
+20,L16,1,96,20161115,526.813
+21,Mock,1,96,20161115,-286.388
+22,Mock,2,96,20161115,-49.1675
+23,L1,2,96,20161115,3908.95
+24,L2,2,96,20161115,-264.671
+25,L4,2,96,20161115,364.933
+26,L6,2,96,20161115,-84.9508
+27,L7,2,96,20161115,-347.168
+28,L11,2,96,20161115,74.1227
+29,L12,2,96,20161115,-204.713
+30,L16,2,96,20161115,48.9294
+31,PHH,1,96,20161115,621.501
+32,PHH,2,96,20161115,805.512
+33,L17,1,96,20161115,-190.471
+34,L18,1,96,20161115,-307.878
+35,L19,1,96,20161115,-291.054
+36,L20,1,96,20161115,23.3205
+37,L22,1,96,20161115,-282.172
+38,L23,1,96,20161115,-303.39
+39,L24,1,96,20161115,-475.741
+40,L26,1,96,20161115,482.678
+41,PCD,1,96,20161115,1096020
+42,PCD,2,96,20161115,1028630
+43,L17,2,96,20161115,60.8281
+44,L18,2,96,20161115,332.344
+45,L19,2,96,20161115,3863.74
+46,L20,2,96,20161115,2818.24
+47,L22,2,96,20161115,582.08
+48,L23,2,96,20161115,583.117
+49,L24,2,96,20161115,-137.397
+50,L26,2,96,20161115,545.602
+51,Vic,1,96,20161115,216380
+52,Vic,2,96,20161115,91222.6
+,,,,,

```

Comment:
Wouldn't delete it outright, but instead move it into a separate directory, e.g. `ignored_data/`, and keep a structured plain text file that describes why that data is ignored. We can chat in the afternoon about it.